Dependencies/AndroidExtensions/Include/AndroidExtensions/JavaWrappers.h

```diff
@@ -277,6 +291,15 @@ namespace android::view
 
         Display getDefaultDisplay();
     };
+
+    class Surface : public java::lang::Object
+    {
+    public:
+        Surface();
+
+        void initWithSurfaceTexture(android::graphics::SurfaceTexture surfaceTexture);
+        ANativeWindow* getNativeWindow();
```

Comment:
I would remove this since it is not part of the Java API, and the caller can easily do this by just passing the `Surface` directly into `ANativeWindow_fromSurface` since it has an implicit conversion to the underlying `jobject`.